doc/pr2_tutorials/planning/src/move_group_interface_tutorial.cpp

```diff
@@ -140,17 +140,21 @@ int main(int argc, char **argv)
   // Let's set a joint space goal and move towards it.  This will replace the
   // pose target we set above.
   //
-  // First get the current set of joint values for the group.
-  std::vector<double> group_variable_values;
-  group.getCurrentState()->copyJointGroupPositions(group.getCurrentState()->getRobotModel()->getJointModelGroup(group.getName()), group_variable_values);
+  // To start, we'll create an pointer that references the current robot's state.
+  // RobotState is the object that contains all the current position/velocity/acceleration data
+  moveit::core::RobotStatePtr current_state = move_group.getCurrentState();
+  //
+  // Next get the current set of joint values for the group.
+  std::vector<double> joint_group_positions;
+  current_state->copyJointGroupPositions(joint_model_group, joint_group_positions);
 
-  // Now, let's modify one of the joints, plan to the new joint
-  // space goal and visualize the plan.
-  group_variable_values[0] = -1.0;
-  group.setJointValueTarget(group_variable_values);
-  success = group.plan(my_plan);
+  // Now, let's modify one of the joints, plan to the new joint space goal and visualize the plan.
+  joint_group_positions[0] = -1.0;
```

Comment:
Would it make sense to specify something like `-M_PI/4`, so that people realize the joint position is radian in this case? -1 is not your typical radian value.. and I met way too many students who didn't understand radian at all...